File: Python/1066.py
# -*- coding: utf-8 -*-
ls = []
valPar = 0
valImpar = 0
valPo = 0
valNeg = 0
        
ls.append(int(input()))
ls.append(int(input()))
ls.append(int(input()))
ls.append(int(input()))
ls.append(int(input()))

for i in ls:
    if i % 2 == 0:
        valPar += 1
            
    else:
        valImpar += 1
            
    if i > 0:
        valPo += 1

    else:
        if i < 0:
            valNeg += 1
        
print(valPar, "valor(es) par(es)")
print(valImpar, "valor(es) impar(es)")
print(valPo, "valor(es) positivo(s)")
print(valNeg, "valor(es) negativo(s)")

File: Java/1066.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        int val[] = new int[5];
        int valPar = 0;
        int valImpar = 0;
        int valPo = 0;
        int valNeg = 0;
        
        for (int i = 0; i < 5; i++) {
            val[i] = leitor.nextInt();
        }
        
        for (int i = 0; i < 5; i++) {
            if (val[i] % 2 == 0) {
                valPar++;
            }
            if (val[i] % 2 != 0) {
                valImpar++;
            }
            if (val[i] > 0) {
                valPo++;
            }
            if (val[i] < 0) {
                valNeg++;
            }
        }
        
        System.out.println(valPar + " valor(es) par(es)");
        System.out.println(valImpar + " valor(es) impar(es)");
        System.out.println(valPo + " valor(es) positivo(s)");
        System.out.println(valNeg + " valor(es) negativo(s)");
    }
 
}